superset/config.py

```diff
@@ -585,6 +588,13 @@ class CeleryConfig:  # pylint: disable=too-few-public-methods
 # dictionary.
 JINJA_CONTEXT_ADDONS: Dict[str, Callable] = {}
 
+# A dictionary of macro template processors that gets merged into global
+# template processors. The existing template processors get updated with this
+# dictionary, which means the existing keys get overwritten by the content of this
+# dictionary. The customized addons don't necessarily need to use jinjia templating
+# language. This allows you to define custom logic to process macro template.
+CUSTOM_TEMPLATE_PROCESSOR = {}  # type: Dict[str, BaseTemplateProcessor]
```

Comment:
Nit: naming here -- `CUSTOM_TEMPLATE_PROCESSOR` should be something like `CUSTOM_TEMPLATE_PROCESSOR_MAP` or even just `CUSTOM_TEMPLATE_PROCESSORS`, as there can potentially be several processors configured via this config property.

Reply:
Sure!